modules/wildfly/pom.xml

```diff
@@ -51,6 +51,13 @@
             <groupId>org.hibernate.ogm</groupId>
             <artifactId>hibernate-ogm-infinispan</artifactId>
             <optional>true</optional>
+            <exclusions>
+                <exclusion>
+                    <!-- The Lucene dependency is not compatible with Neo4j embedded -->
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-search-engine</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
I'm wondering if it's a good idea to remove that (and the related stuff below - enforcer and so on). The fact that the versions are aligned is just pure luck, isn't it? There's a good chance minor versions might not be aligned if they stick to 5.5.0.